src/test/java/eu/dissco/core/digitalspecimenprocessor/service/MidsServiceTest.java

```diff
@@ -35,266 +155,4 @@ void testCalculateMids(DigitalSpecimen digitalSpecimen, int midsLevel) {
     // Then
```

Comment:
This is amazing. Testing the entire class logic with a 2 line test. 10/10

Reply:
Well yes but still we need 141 lines for preparing the two lines :wink:

Reply:
not important